src/jade/app/fetch.py

```
@@ -44,16 +48,64 @@ def _fetch_from_git(url: str, authorization_token: str | None = None) -> str | b
+    authorization_token : str, optional
+        Authorization token to access the IAEA repository. Default is None.
+    Returns
+    -------
+    extracted_folder: Pathlike | bool
+        path to the extracted folder. False if the download was not successful.
+    """
+    gl = gitlab.Gitlab(url=url, private_token=authorization_token, ssl_verify=False)
+    try:
+        gl.auth()
+    except gitlab.exceptions.GitlabAuthenticationError:
+        logging.error("Gitlab authentication failed")
```

Ruff (0.13.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)